director/extensions.py

```diff
@@ -98,6 +102,85 @@ def init_app(self, app):
         self.conf.update(app.config.get("CELERY_CONF", {}))
 
 
+# Sentry Extension
+class DirectorSentry:
+    def __init__(self):
+        self.app = None
+
+    def init_app(self, app):
+        self.app = app
+
+        if self.app.config["SENTRY_DSN"]:
```

Comment:
May be we should update also this list in: `director/settings.py`
```
HIDDEN_CONFIG = [
    "DIRECTOR_ENABLE_DARK_THEME",
    "DIRECTOR_API_URL",
    "DIRECTOR_FLOWER_URL",
    "DIRECTOR_DATABASE_URI",
    "DIRECTOR_DATABASE_POOL_RECYCLE",
    "DIRECTOR_BROKER_URI",
    "DIRECTOR_RESULT_BACKEND_URI",
]
```

Reply:
done